### PR TITLE
SEC04-AWS100

### PR DESCRIPTION
**Pull request by syedautherabbas**
SEC04-AWS100 — Create an IAM user

### CODE REVIEW

File: Projects/SEC/SEC04/SEC04-AWS100.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+
+# SEC04-AWS100 — Create an IAM user
+
+## Cloud Service Provider
+
+* Amazon Web Services
+
+## Difficulty
+Level 100 (Introductory)
```
Comment (what-name) [inline Projects/SEC/SEC04/SEC04-AWS100.md]: - Please use bullet point

File: Projects/SEC/SEC04/SEC04-AWS100.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+## Estimated time:
+ 1 hour
```
Comment (what-name) [inline Projects/SEC/SEC04/SEC04-AWS100.md]: - Please use bullet point

File: Projects/SEC/SEC04/SEC04-AWS100.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+## Project's Author(s)
+* [Syed Auther](https://twitter.com/syedauther)
+
+## Objectives
+
+### You need to complete the following:
+
+* Create a new user in AWS IAM with programmatic and console access
+* Add user to Admin group
```
Comment (what-name) [inline Projects/SEC/SEC04/SEC04-AWS100.md:22]: - "Admin group" does not provide enough information for a Lvl100 project. Please describe this in detail, first time IAM user creation is not a walk in the park (as much as it seems like so to us now).
- I'd recommend creating three bullet points in the form of:
  - Create a new Group
  - Add 'AdministratorAccess' privileges to the newly created group
  - Add new IAM user to that group

I see there are link provided down below, but it's still not easy to understand that these resources have to be created seperately.

File: Projects/SEC/SEC04/SEC04-AWS100.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+* Apply an IAM password policy with best security practises.
+
+
+
+### You need to answer the following: 
+
+* What is Identity and Access Management? 
+* What is a root user?
+* How is a root user different from an Admin user? 
+* What is console access and Programmatic access?
```
Comment (what-name) [inline Projects/SEC/SEC04/SEC04-AWS100.md]: - No need for capital "P"rogrammatic

File: Projects/SEC/SEC04/SEC04-AWS100.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+* What is a root user?
+* How is a root user different from an Admin user? 
+* What is console access and Programmatic access? 
+* What is the access key and secret key? 
+* What is MFA for root user? Why is it a good practise to configure MFA? 
+* What are policies and how to create them?
+* What are roles and how to create them?
+* What is the difference between a role and a policy?
+* What is a user group? 
+* What are some good security practises for password policies?
```
Comment (what-name) [inline Projects/SEC/SEC04/SEC04-AWS100.md]: - Typo: practices

File: Projects/SEC/SEC04/SEC04-AWS100.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+* [Syed Auther](https://twitter.com/syedauther)
+
+## Objectives
+
+### You need to complete the following:
+
+* Create a new user in AWS IAM with programmatic and console access
+* Add user to Admin group 
+* Enable MFA for the IAM root user 
+* Apply an IAM password policy with best security practises.
```
Comment (what-name) [inline Projects/SEC/SEC04/SEC04-AWS100.md]: - Typo: practices (practise is a verb)

File: Projects/SEC/SEC04/SEC04-AWS100.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+* [Policy examples](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_examples.html)
+* [IAM Roles](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html)
+* [IAM Best practices](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html)
+* [Creating an Admin user](https://docs.aws.amazon.com/IAM/latest/UserGuide/getting-started_create-admin-group.html)
+* [Setting a password policy](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_passwords_account-policy.html?icmpid=docs_iam_console)
+
+
+## Tips
+* User [Google Authenticator ](https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2) for MFA.
+* Do not store AWS access keys you create with this task on public platforms like Github
```
Comment (what-name) [inline Projects/SEC/SEC04/SEC04-AWS100.md]: - I think this should be phrased with much more power, it happens way too often (especially with beginners, I've almost been there😅)

Maybe: Do not ever commit, add or push any of your access and secret keys to source control systems like git. If it ever happens, make sure to delete the keys immediately in your console.

I'm def open to suggestions tho

**Review by what-name — CHANGES_REQUESTED** [review]
Mainly some minor/visual stuff. Everything else is great! This is a much needed project👍

File: Projects/SEC/SEC04/SEC04-AWS100.md (hunk 3 above)
Comment (what-name) [inline Projects/SEC/SEC04/SEC04-AWS100.md]: - "Admin group" does not exist as such. Creating users for the very first time in IAM is not a walk in the park, as much as it seems like it to us now. Please explain this step in detail.
  - Create a new group called "Admins"
  - Add the `AdministratorAccess` managed policy to the new group
  - Add your new user to the "Admins" group

**Comment by what-name** [comment]
Add:
### Cost
- IAM is a free service

File: Projects/SEC/SEC04/SEC04-AWS100.md (hunk 6 above)
Comment (syedautherabbas) [reply Projects/SEC/SEC04/SEC04-AWS100.md]: I feel so dumb now!  #100daysofGrammar haha thanks @what-name !

File: Projects/SEC/SEC04/SEC04-AWS100.md (hunk 7 above)
Comment (syedautherabbas) [reply Projects/SEC/SEC04/SEC04-AWS100.md]: Agreed. Updated the text to add more caution.

File: Projects/SEC/SEC04/SEC04-AWS100.md (hunk 3 above)
Comment (syedautherabbas) [reply Projects/SEC/SEC04/SEC04-AWS100.md]: Oops! You are right. Fixed this as well as subheaders under the Add user to Admin group header.

File: Projects/SEC/SEC04/SEC04-AWS100.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+* [Root user in AWS](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_root-user.html)
+* [Policy examples](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_examples.html)
+* [IAM Roles](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html)
+* [IAM Best practices](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html)
+* [Creating an Admin user](https://docs.aws.amazon.com/IAM/latest/UserGuide/getting-started_create-admin-group.html)
+* [Setting a password policy](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_passwords_account-policy.html?icmpid=docs_iam_console)
+
+
+## Tips
+* User [Google Authenticator ](https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2) for MFA.
```
Comment (what-name) [inline Projects/SEC/SEC04/SEC04-AWS100.md]: Typo: use(-r). Just caught this.

File: Projects/SEC/SEC04/SEC04-AWS100.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+
+## Objectives
+
+### You need to complete the following:
+
+* Create a new user in AWS IAM with programmatic and console access
+* Add user to Admin group 
+--Create a new group called "Admins"
+--Add the AdministratorAccess managed policy to the new group
+--Add your new user to the "Admins" group
```
Comment (what-name) [inline Projects/SEC/SEC04/SEC04-AWS100.md]: These subheadings are not working Markdown.
You can do subheadings with indentation:

```
* Add user to an Admin group
  * Create a new group called "Admins"
  * Add the `AdministratorAccess` managed policy to the new group
  * Add your new user to the "Admins" group
```
⬇️
* Add user to an Admin group
  * Create a new group called "Admins"
  * Add the `AdministratorAccess` managed policy to the new group
  * Add your new user to the "Admins" group

**Pull request merged**